### PR TITLE
Update navbar links

### PR DESCRIPTION
**Pull request by mauricio-prl**
References: https://github.com/rubyforgood/babywearing/issues/82

### Description
This PR is updating the navbar, doing that only admins and volunteers can be able to see the create user link.

### How Has This Been Tested?
I wrote some feature tests with rspec to ensure this functionality.

### CODE REVIEW

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -9,8 +9,10 @@
     <div class="collapse navbar-collapse" id="navbarSupportedContent">
       <ul class="navbar-nav mr-auto">
         <li class="nav-item active background-blue">
-          <%= link_to new_user_registration_path, class: 'nav-link nav-icon nav-first' do %>
-            <%= image_tag "BW- CREATE USER ICON.svg", alt:"Create a User" %><p>CREATE USER</p>
+          <% if current_user&.role == 'admin' || current_user&.role == 'volunteer' %>
```
Comment (dixpac) [inline app/views/layouts/_header.html.erb]: You can use `current_user.admin? || current_user.volunteer?`  Rails enum gives us these nice helper methods :)

File: spec/features/admins_and_volunteers_can_create_member_spec.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+require 'rails_helper'
+
+RSpec.describe 'only admins and volunteers can see the create user' do
+  let(:admin) { users(:admin) }
+  let(:volunteer) { users(:volunteer) }
+  let(:member) { users(:member) }
+
+  scenario 'when user is an admin or a volunteer' do
+    [admin, volunteer].each do |user|
```
Comment (dixpac) [inline spec/features/admins_and_volunteers_can_create_member_spec.rb:11]: 👍

**Review by dixpac — CHANGES_REQUESTED** [review]
Awesome just a tinny comment

File: app/views/layouts/_header.html.erb (hunk 1 above)
Comment (dixpac) [reply app/views/layouts/_header.html.erb]: or we could reuse this helper https://github.com/rubyforgood/babywearing/blob/master/app/helpers/users_helper.rb#L8

Comment (mauricio-prl) [reply app/views/layouts/_header.html.erb]: Very nice! Thank you!

Comment (mauricio-prl) [reply app/views/layouts/_header.html.erb]: Well I had to add the safe navigation in your helper method, can it be a future problem?

**Review by dixpac — APPROVED** [review]
👍 🥇

File: spec/features/admins_and_volunteers_can_create_member_spec.rb [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      sign_in user
+      visit(root_path)
```
Comment (renatamarques97) [inline spec/features/admins_and_volunteers_can_create_member_spec.rb:13]: you can put this in a "before"

File: spec/features/admins_and_volunteers_can_create_member_spec.rb [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+      expect(page).to have_link('CREATE USER')
+    end
+  end
+
+  scenario 'when user is a member' do
```
Comment (renatamarques97) [inline spec/features/admins_and_volunteers_can_create_member_spec.rb:19]: use "context" instead of "scenario"

**Pull request merged**